info_db.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import sqlite3
+from pathlib import Path
+from sqlite3 import Error
+
+Path("info_database.db")
```

Comment:
Where are you going to store the `Path` ? How do you want to use it in code below ?